.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+; Ignore any non 'sites' code;
+omit =
+    webapp/shop/*
```

Comment:
webapp/certified seems like a grey area, I'm not sure if it should be here too. Andrej and his team have owned updates to this for the last year, but is that the same as not being 'sites' code? Might be a dumb question sorry 😅